### DISCUSSION TITLE
Win rate should be reset when changing Tier

### DISCUSSION THREAD

**Discussion by Nettogrof** [BattlesnakeOfficial/feedback#81] [Feature Requests]
During a league (lie summer league) it would be nice that the win rate were about in win rate in the current snake tier.

Currently Pretzel join the summer later than most of the snake, so Pretzel had to beat a lot of low tier snake to reach the top, making the win rate  irrevelant if compare to Pruzze one.

**Comment by bvanvugt** [comment]
Yeah, this is a super good suggestion. Not sure we'll be able to make this happen before Fall League, but it's definitely something we'd like to do.
